src/test/e2e/13_find_images_test.go

```diff
@@ -93,9 +57,32 @@ func TestFindImages(t *testing.T) {
 		path := filepath.Join("src", "test", "packages", "00-find-images-with-vars")
 		stdOut, _, err = e2e.Zarf("prepare", "find-images", path, "--deploy-set", "BUSYBOX_IMAGE=busybox:earliest")
 		require.NoError(t, err)
-
 		require.Contains(t, stdOut, "nginx:latest", "Manifests aren't interpreting vars")
 		require.Contains(t, stdOut, "busybox:earliest", "Values files aren't interpreting vars")
 	})
 
+	t.Run("zarf test find images --why  w/ helm chart success", func(t *testing.T) {
+		t.Parallel()
+
+		testPackagePath := filepath.Join("examples", "wordpress")
+		stdOut, _, err := e2e.Zarf("dev", "find-images", testPackagePath, "--why", "docker.io/bitnami/apache-exporter:0.13.3-debian-11-r2")
+		require.NoError(t, err)
+		require.Contains(t, stdOut, "component: wordpress")
+		require.Contains(t, stdOut, "chart: wordpress")
+		require.Contains(t, stdOut, "image: docker.io/bitnami/wordpress:6.2.0-debian-11-r18")
+	})
+
+	t.Run("zarf test find images --why w/  manifests success", func(t *testing.T) {
+		t.Parallel()
+
+		testPackagePath := filepath.Join("examples", "manifests")
+
+		stdOut, _, err := e2e.Zarf("dev", "find-images", testPackagePath, "--why", "httpd:alpine3.18")
+		require.NoError(t, err)
+		require.Contains(t, stdOut, "component: httpd-local")
+		require.Contains(t, stdOut, "manifest: simple-httpd-deployment")
+		require.Contains(t, stdOut, "image: httpd:alpine3.18")
```

Comment:
Commenting on this - this is not a particularly great test now - this could be repeated many times or other issues could arise - we can talk about how to handle this, but it is a regression in coverage

Reply:
Might be worth a `tech-debt` issue